example/controller-registration.yaml

```diff
@@ -17,8 +17,7 @@ metadata:
 spec:
   deployment:
     deploymentRefs:
-      - name: shoot-lakom-service
-    policy: Always
+    - name: shoot-lakom-service
   resources:
-    - kind: Extension
-      type: shoot-lakom-service
+  - kind: Extension
+    type: shoot-lakom-service
```

Comment:
Trimming these white spaces seems to be related to the yq used here https://github.com/gardener/gardener-extension-shoot-lakom-service/blob/02c395c88cc5bc8f939ad50c568142c296ccedf9/hack/set-controller-deployment-policy-to-always.sh#L14. 

I just double checked, and the script is using the `yq` installed in the `hack/tools/bin`. The change was interoduced with f16753958fce80fa4bd1f8e89f9cc2a3af2f1b34 this morning, can you check if something went wrong on your machine?